src/test/java/com/gooddata/http/client/GoodDataHttpClientIntegrationTest.java

```diff
@@ -134,23 +138,90 @@ public void getProjectOkloginAndTtRefresh() throws IOException {
+                .withEncoding(Charset.forName("UTF-8"))
+                .withContentType("application/json; charset=UTF-8");
+
+        onRequest()
+                .havingMethodEqualTo("GET")
+                .havingURIEqualTo(GDC_PROJECTS_URL)
+                .havingHeaderEqualTo("Accept", "application/json")
+                .havingHeaderEqualTo("X-GDC-AuthTT", "cookieTt")
+        .respond()
+                .withStatus(200)
+                .withBody("{\"about\":{\"summary\":\"Project Resources\",\"category\":\"Projects\",\"links\":[]}}")
+                .withEncoding(Charset.forName("UTF-8"))
+                .withContentType("application/json; charset=UTF-8");
+
+        onRequest()
+                .havingMethodEqualTo("GET")
+                .havingURIEqualTo(GDC_TOKEN_URL)
+        .respond()
+                .withStatus(401)
+                .withHeader("WWW-Authenticate", "GoodData realm=\"GoodData API\" cookie=GDCAuthSST")
+                .withBody("{\"parameters\":[],\"component\":\"Account::Token\",\"message\":\"/gdc/account/login\"}")
+                .withEncoding(Charset.forName("UTF-8"))
+                .withContentType("application/json");
+        onRequest()
+                .havingMethodEqualTo("GET")
+                .havingURIEqualTo(GDC_TOKEN_URL)
+                .havingHeaderEqualTo(SST_HEADER, "cookieSst")
+        .respond()
+                .withStatus(200)
+                .withBody("{ \"userToken\" : { \"token\" : \"cookieTt\" } }")
+                .withHeader("Set-Cookie", "GDCAuthTT=cookieTt; path=/gdc; secure; HttpOnly")
+                .withEncoding(Charset.forName("UTF-8"))
+                .withContentType("application/json");
+
+        onRequest()
+                .havingMethodEqualTo("POST")
+                .havingURIEqualTo(GDC_LOGIN_URL)
+                .havingHeaderEqualTo("Accept", "application/json; charset=UTF-8")
+                .havingBodyEqualTo("{\"postUserLogin\":{\"login\":\"user@email.com\",\"password\":\"top secret\",\"remember\":0,\"verify_level\":2}}")
+        .respond()
+                .withStatus(200)
+                .withBody("{\"userLogin\":{\"profile\":\"/gdc/account/profile/asdfasdf45t4ar\",\"token\":\"cookieSst\",\"state\":\"/gdc/account/login/asdfasdf45t4ar\"}}")
+                .withContentType("application/json")
+                .withEncoding(Charset.forName("UTF-8"));
+
+        final HttpClient httpClient = HttpClientBuilder.create().build();
+        final SSTRetrievalStrategy sstStrategy = new LoginSSTRetrievalStrategy(jadlerLogin, jadlerPassword);
+        final HttpClient client = new GoodDataHttpClient(httpClient, jadlerHost, sstStrategy);
+
+        performGet(client, jadlerHost, REDIRECT_URL, HttpStatus.SC_OK);
```

Comment:
Why? Although it is possible to verify a stubbed invocation, usually it's just redundant.